catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/PiaimcAttribute.java

```diff
@@ -25,22 +25,185 @@
 import org.codice.imaging.nitf.core.tre.Tre;
 
 import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.AttributeType;
+import ddf.catalog.data.impl.BasicTypes;
 
 /**
  * TRE for "Profile for Imagery Access Image"
  */
-class PiaimcAttribute extends NitfAttributeImpl<Tre> {
+public class PiaimcAttribute extends NitfAttributeImpl<Tre> {
 
     private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
-    
+
     public static final String CLOUDCVR_NAME = "CLOUDCVR";
 
-    public static final PiaimcAttribute CLOUDCVR = new PiaimcAttribute(Isr.CLOUD_COVER,
+    public static final String SRP_NAME = "SRP";
+
+    public static final String SENSMODE_NAME = "SENSMODE";
+
+    public static final String SENSNAME_NAME = "SENSNAME";
+
+    public static final String SOURCE_NAME = "SOURCE";
+
+    public static final String COMGEN_NAME = "COMGEN";
+
+    public static final String SUBQUAL_NAME = "SUBQUAL";
+
+    public static final String PIAMSNNUM_NAME = "PIAMSNNUM";
+
+    public static final String CAMSPECS_NAME = "CAMSPECS";
+
+    public static final String PROJID_NAME = "PROJID";
+
+    public static final String GENERATION_NAME = "GENERATION";
+
+    public static final String ESD_NAME = "ESD";
+
+    public static final String OTHERCOND_NAME = "OTHERCOND";
+
+    public static final String MEANGSD_NAME = "MEANGSD";
+
+    public static final String IDATUM_NAME = "IDATUM";
+
+    public static final String IELLIP_NAME = "IELLIP";
+
+    public static final String PREPROC_NAME = "PREPROC";
+
+    public static final String IPROJ_NAME = "IPROJ";
+
+    public static final String SATTRACK_PATH_NAME = "SATTRACK_PATH";
+
+    public static final String SATTRACK_ROW_NAME = "SATTRACK_ROW";
+
+    private static final String ATTRIBUTE_NAME_PREFIX = "piaimc.";
+
+    static final PiaimcAttribute CLOUDCVR = new PiaimcAttribute(Isr.CLOUD_COVER,
```

Comment:
I believe we want to duplicate this field, so we have the normalized attribute as well as the original.